src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorActor.scala

```diff
@@ -292,9 +287,11 @@ class ClusterMonitorActor(val cluster: Cluster,
         case Running if leoClusterStatus == Starting && runningInstanceCount == googleInstances.size =>
           getMasterIp.flatMap {
             case Some(ip) =>
-              isProxyAvailable(leoClusterStatus, ip).map {
-                case true =>  ReadyCluster(ip, googleInstances)
-                case false => NotReadyCluster(ClusterStatus.Running, googleInstances)
+              dbRef.inTransaction { _.clusterQuery.updateClusterHostIp(cluster.id, Some(ip)) }.flatMap { _ =>
```

Comment:
Why do we need to update the ip in the database here? I think `clusterQuery.setToRunning` already does that.

Reply:
It turns out, we need to update the database because later on we rely on the database to get the host information to determine `ifProxyAvailable`, or else, we think that the host isn't available and get stuck checking the status. This happens when we resume clusters and the cluster transitions from `Starting` to `Running`.